app/assets/stylesheets/application.css.scss

```diff
@@ -63,6 +63,25 @@ footer {
   padding: 10px;
 }
 
+.splash-title {
+  font-family: 'Miltonian Tattoo', cursive;
+  margin: 0;
+  text-align: center;
+}
+
+.splash-body {
+  font-family: 'Miltonian Tattoo', cursive;
+  text-align: center;
+
+  a {
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.